Frontend/Pages/Home/Index.cshtml.cs

```diff
@@ -3,6 +3,7 @@
 using Data;
 using Data.Models;
 using Frontend.ExtensionMethods;
+using Microsoft.AspNetCore.Authorization;
```

Comment:
Prob not needed but we can just remove it another time

Reply:
Thanks, removed